site/content/docs/groups/_index.md

```diff
@@ -13,37 +13,64 @@ aliases:
 
 ---
 
-Groups allow you to organize your checks and centralize settings like base URL's, headers, variables and other properties
+Groups allow you to organize your checks and centralize settings like base URLs, headers, variables and other properties
 a collection of checks can share. 
 
+![Check group screenshot](/docs/images/groups/group-in-dashboard.png)
+
 Example use cases for groups are organizing your checks around:
 
 - A common URL endpoint
 - A specific team
 - A specific feature in your app
 - A test suite; trigger all checks after a deployment
 
-> When using groups, any group-level configurations such as the runtime, activated & muted state, tags, and alert channels are inherited by all Checks in the group. 
+> Group-level configurations, such as the runtime, activated & muted state, tags, and alert channels, will override check-level configurations. 
 
-## Key features
+> [!WARNING]
+> Alert settings being controlled at group level means that a check that is part of a group that has no connected alert channels *will not alert*.
 
-The screenshot below gives a quick overview of the groups' key features.
+## Creating a check group
 
 ![Check group editor screenshot](/docs/images/groups/group-editor.png)
 
-1. **Activate and mute** all checks in a group
-2. Configure **shared settings** for all checks in the group
-    1. API check defaults like headers, assertions and setup & teardown scripts
-    2. Environment variables
-    3. Data center locations
-    4. Alert setting and alert channels
-    5. CI/CD triggers
-3. **Run all checks in one go** with a configurable concurrency.
-4. Tweak checks in the inline "mini editor" to **quickly build up a group of similar checks**
-5. Use a **common base URL** for your API checks    
+### Name and tags
+Pick a meaningful name for your group. A meaningful name will not only help you and others identify your group within Checkly, but it will help provide a better alerting experience if checks in this group fall into an alert state. Tags can relate your groups together, they also determine which checks are shown on your public [dashboards](/docs/dashboards/).  
+
+### Checks
+Add new or existing checks to this group. If you add an existing check, the group configuration will overwrite certain check configurations, like run locations, retries, and alerting. 
+
+For example, if you create a check that runs in `eu-west-1` but then add it to a group running in `us-east-1`, then the group settings will take precedence and overwrite the individual check locations. 
```

Comment:
@guolau since we are bringing up an example, let's finish it instead of switching back over to a generic answer: if you create a check that runs in `eu-west-1` but then add it to a group running in `us-east-1`, then the check will run from `us-east-1` only.